<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubmodulesCreateEditDlg</class>
 <widget class="QDialog" name="SubmodulesCreateEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Submodule</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QComboBox" name="cboIgnoreStrategy">
        <item>
         <property name="text">
          <string notr="true">None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Dirty</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Untracked</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">All</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Current SHA-1</string>
        </property>
        <property name="buddy">
         <cstring>txtSHA1</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ignore strategy</string>
        </property>
        <property name="buddy">
         <cstring>cboIgnoreStrategy</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cboUpdateStrategy">
        <item>
         <property name="text">
          <string notr="true">Checkout</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Rebase</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Merge</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Update strategy</string>
        </property>
        <property name="buddy">
         <cstring>cboUpdateStrategy</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>txtName</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="chlFetchRecursive">
        <property name="text">
         <string>Fetch recursive</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="txtSHA1">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtName"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkHasPath">
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtPath">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Url</string>
        </property>
        <property name="buddy">
         <cstring>txtUrl</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtUrl"/>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>chkHasPath</tabstop>
  <tabstop>txtPath</tabstop>
  <tabstop>txtUrl</tabstop>
  <tabstop>cboIgnoreStrategy</tabstop>
  <tabstop>cboUpdateStrategy</tabstop>
  <tabstop>chlFetchRecursive</tabstop>
  <tabstop>txtSHA1</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SubmodulesCreateEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SubmodulesCreateEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkHasPath</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtPath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
